django_sharding_library/fields.py

```diff
@@ -129,7 +130,10 @@ def __init__(self, *args, **kwargs):
         if type(model_class) == str:
             app_label = model_class.split('.')[0]
             app = apps.get_app_config(app_label)
-            model_class = app.get_model(model_class[len(app_label) + 1:])
+            if (1, 11) > django.VERSION:
+                model_class = app.get_model(model_class[len(app_label) + 1:])
+            else:
+                model_class = app.get_model(model_class[len(app_label) + 1:], require_ready=False)
```

Comment:
In Django 1.11 `django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet.` is raised

https://docs.djangoproject.com/en/1.11/ref/applications/#django.apps.AppConfig.get_model